lighthouse-core/test/gather/mock-commands.js

```diff
@@ -63,7 +63,7 @@ function createMockSendCommandFn(options) {
 
       const indexOfResponse = mockResponses
         .findIndex(entry => entry.command === command && entry.sessionId === sessionId);
-      if (indexOfResponse === -1) throw new Error(`${command} unimplemented`);
+      if (indexOfResponse === -1) return Promise.reject(new Error(`${command} unimplemented`));
```

Comment:
should we just make this function async?

Reply:
done 👍